### DISCUSSION TITLE
no function

### DISCUSSION THREAD

**Discussion by BerndSteinig** [aclap-dev/video-downloadhelper#866] [Help and Q&A] [answered]
### Your question

I actually wanted to buy this program. Unfortunately, it doesn't keep its promises. When I start a video on Facebook, it doesn't appear in the list, if it appears at all. Sometimes I have consecutive videos with the same picture of me and sequences of different lengths, lasting only seconds, which are displayed. I have installed the additional tool. What could be the problem now?

### Addon & CoApp version and details

all the newest version

**Comment by Mattmjs** [comment, marked as answer]
It has never worked properly on Facebook, there has been a trick to make the link switch to the mobile interface.
Replacing www with m is what you should do. That is the only workable solution.

**Reply by BerndSteinig** [reply]
thx for the super help.
